### PR TITLE
Langing page added

### PR DESCRIPTION
**Pull request by pawelchabros**
App deployed [here](https://connect.appsilon.com/content/efd36537-589d-4106-a117-891250c84054).

### CODE REVIEW

File: inst/examples/showcase/app.R [hunk 1]
```diff
@@ -153,7 +158,38 @@ makeRouter <- function(items) {
...
+          ),
+          " app built in Shiny"
+        ),
+        span(class = "font-mono", "shiny.react + Blueprint = shiny.blueprint"),
+        Card(
+          class = "home-page-section",
+          span("Welcome to ", tags$strong(class = "font-mono", "shiny.blueprint"), " demo!"),
+          span(
+            tags$strong(class = "font-mono", "shiny.blueprint"), " is a package that allows ",
+            "you to build Shiny apps using Bleprint - a React-based UI toolkit for the web."
```
Comment (averissimo) [inline inst/examples/showcase/app.R:183]: Typo on Bleprint

```suggestion
            "you to build Shiny apps using Blueprint - a React-based UI toolkit for the web."
```

File: inst/examples/showcase/app.R [hunk 2]
```diff
@@ -153,7 +158,38 @@ makeRouter <- function(items) {
...
+          ),
+          span("Use the menu on the left to explore live demos of all available components.")
+        )
```
Comment (averissimo) [inline inst/examples/showcase/app.R:186]: [optional] add icons for the 2 libraries

```suggestion
        ),
        Card(
          class = "home-page-section",
          span(
            tags$strong(class = "font-mono", "Shiny.react"),
            " R package enables using React in Shiny apps.",
            "It contains R and JS code which is independent from the React library that is being wrapped."
          ),
          a(
            href = "https://appsilon.github.io/shiny.react/",
            target = "_blank",
            img(
              class = "logo",
              src = "https://github.com/Appsilon/shiny.react/raw/master/man/figures/shiny-react.png"
            )
          )
        ),
        Card(
          class = "home-page-section",
          span(
            tags$strong(class = "font-mono", "Blueprint"),
            " is a React-based UI toolkit for the web.",
            "It is optimized for building complex, data-dense web interfaces for desktop applications",
            "which run in modern browsers and IE11. This is not a mobile-first UI toolkit."
          ),
          a(
            href = "https://blueprintjs.com/",
            target = "_blank",
            img(
              class = "logo",
              src = "https://cloud.githubusercontent.com/assets/464822/20228152/d3f36dc2-a804-11e6-80ff-51ada2d13ea7.png"
            )
          )
        )
```

File: inst/examples/showcase/static/css/styles.css [hunk 3]
```diff
@@ -1,9 +1,67 @@
...
+  display: flex;
+  flex-direction: column;
+  gap: 15px;
+  padding: 25px;
+  margin-top: 35px;
+}
+
+.font-mono {
+  font-family: monospace;
 }
```
Comment (averissimo) [inline inst/examples/showcase/static/css/styles.css:66]: [optional] if you add the new cards, the logo size needs to be smaller than the originals

```suggestion

.logo {
  height: 100px;
}
```

File: inst/examples/showcase/app.R [hunk 4]
```diff
@@ -163,7 +199,7 @@ addResourcePath("showcase-static", "./static")
 shinyApp(
   ui = tagList(
     tags$script(
-      src = "showcase-static/js/highlight.min.js"
+      src = "showcase-static/js/highlight.v11.7.0.min.js"
```
Comment (averissimo) [inline inst/examples/showcase/app.R:202]: [question, optional]
have you considered using a CDN?

This doesn't add that much size to the js so it's completely fine like this

**Review by averissimo — CHANGES_REQUESTED** [review]
The typo needs changing, all other comments are optional

**Review by averissimo — APPROVED** [review]
I should've approved immediately.

The only change I think is required is the typo 😁 (but there's no need for a follow-up)

File: inst/examples/showcase/app.R (hunk 1 above)
Comment (pawelchabros) [reply inst/examples/showcase/app.R:183]: Fixed :+1:

File: inst/examples/showcase/app.R (hunk 4 above)
Comment (pawelchabros) [reply inst/examples/showcase/app.R:202]: Previously Filip also downloaded source code of `prism.js` so I just automatically did the same.

File: inst/examples/showcase/app.R (hunk 2 above)
Comment (pawelchabros) [reply inst/examples/showcase/app.R:223]: Cards with description and logos added

**Pull request merged**